build.gradle

```diff
@@ -1,15 +1,17 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.4.0-alpha7'
+        classpath 'com.android.tools.build:gradle:3.1.1'
```

Comment:
Please rebase on the latest branch to get the most recent versions of dependencies.